### DISCUSSION TITLE
Possible issues

### DISCUSSION THREAD

**Discussion by huskylogic** [microsoft/winget-cli#3739] [General] [answered]
I've been having trouble with winget and noticed a couple of things.  I manage about 12 computers in my nonprofit, some are in the same location, and others are remote machines. These are what I have questions about. 

1. I get a message: "No installed package found matching input criteria." Is that winget speak for nothing needs to be updated, or does that mean the sources are acting up again and I need to reset everything? The command I'm running is winget upgrade  --all --include-unknown. Keep in mind I ran this on all 12 machines, and two of them didn't give me that error, but they also found packages that needed to be updated. 
2. When I do winget list I see a column for "Name", "ID," "Version," and "Source."  I don't see one for "Available." Is that because I am having issues, or simply because there are no updates available, thus the column is hidden? 

I'm coming from Chocolatey, so I'm aware of how to use the command line, but I'm still learning how to use winget. Any advice you can give me would be helpful.

**Comment by Trenly** [comment, marked as answer]
> Is that winget speak for nothing needs to be updated

This is correct; No updates were found

> ... or simply because there are no updates available, thus the column is hidden?

This is also correct; The column is only shown if new versions are available

**Reply by huskylogic** [reply]
So there isn't anything wrong with my widget that needs to be fixed. It's just them telling me I have nothing to upgrade. When there is something to boost, I will get a new answer. Thank you.
